libraries/redshop/src/Product/Accessory.php

```diff
@@ -54,10 +54,6 @@ public static function getPrice(
                 $accessoryPriceVAT = \RedshopHelperProduct::getProductTax($productId, $accessoryPrice, $userId);
             }
 
-            if ($accessoryMainPrice > 0) {
-                $accessoryMainPriceVAT = \RedshopHelperProduct::getProductTax($productId, $accessoryMainPrice, $userId);
-            }
-
             // Add VAT to accessory prices
             $accessoryPrice     += $accessoryPriceVAT;
             $accessoryMainPrice += $accessoryMainPriceVAT;
```

Comment:
this line and the line declare $accessoryMainPriceVAT above are not necessary anymore

Reply:
@NguyenBao10 done :D